purpl3_rms/src/components/computers/EditComputer.js

```diff
@@ -34,9 +26,7 @@ const EditComputer = (props) => {
 			else{
 				setAdmin(false);
```

Comment:
why is setting the admin to false the else...

Reply:
Ask Daniela lol i didnt write that part